packages/microsite/src/hydrate.tsx

```diff
@@ -10,39 +10,50 @@ export interface HydrationProps {
   fallback?: VNode<any> | null;
 }
 
+const encode = (str: string) => Buffer.from(str).toString("base64");
```

Comment:
I was wondering if this line is removed in the prod bundle and it seems that it is. Rollup is smart :)